guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>In cases of a univariate timeseries, the single variable will be used to forecast for the next given time period, which means the same variable will be used to predict future terms of that variable, and thus has no other explanatory variables. </p><p>In cases of a multivariate time series model, there will be a list of explanatory variables created as follows: &nbsp;making a list of the explanatory variables </p><p>                X = ["Field_1", ("Field_2", True),......."Field_n"] </p><p>Here, Field_2 is treated as categorical, while the others are treated as continuous variables. </p><p>Once the variables are identified, the preprocessing of the data is carried out by the <code>prepare_tabulardata</code> method. This function will take either a non spatial dataframe, a feature layer, or a spatial dataframe containing the dataset as input and will return a TabularDataObject that can be fed into the model for training. The function also works on a combination of feature layers or spatial dataframes with rasters in which the values of the raster corresponding to the features are selected for preparing the final data. &nbsp;The function can also be used with a series of rasters where the dependent variable identifies the target raster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>